package.json

```diff
@@ -58,6 +58,7 @@
     "onCommand:maven.plugin.execute",
     "onCommand:maven.project.addDependency",
     "onCommand:maven.project.excludeDependency",
+    "onCommand:maven.project.jumpToDefinition",
```

Comment:
Is it necessary here? In which case do you activate extension by running this command?

Reply:
What about `excludeDependnecy` and `setVersionto`? I think they are also unnecessary.

Reply:
it depends on whether it's available in command palette. If not, you should remove it.